index.html

```diff
@@ -30,8 +30,11 @@ <h2>A gallery of unique photos.</h2>
 				<div class="inner">
 					<div class="columns">
 							<div class="image fit">
-								<a href="hello-page.html"><img src="images/architecture.jpg" alt="" /></a>
-							</div>							
+								<a href="Marinas.html"><img src="images/architecture.jpg" alt="" /></a>
```

Comment:
```suggestion
								<a href="hello-page.html"><img src="images/architecture.jpg" alt="" /></a>
```

add your photo but don't modify existing photos and links